src/com/lilithsthrone/game/dialogue/utils/EnchantmentDialogue.java

```diff
@@ -418,4 +444,296 @@ public static void initModifiers(AbstractCoreItem ingredient) {
+			
+			Element enchantment = doc.createElement("enchantment");
+			doc.appendChild(enchantment);
+			
+			Element itemEffects = doc.createElement("itemEffects");
+			enchantment.appendChild(itemEffects);
+			
+			for(ItemEffect effect : effects)
+			{
+				effect.saveAsXML(itemEffects, doc);
+			}
+			
+			// Ending stuff:
+			
+			TransformerFactory tf = TransformerFactory.newInstance();
+			Transformer transformer1 = tf.newTransformer();
+			transformer1.setOutputProperty(OutputKeys.OMIT_XML_DECLARATION, "yes");
+			StringWriter writer = new StringWriter();
+
+			transformer1.transform(new DOMSource(doc), new StreamResult(writer));
+			
+			// Save this xml:
+			TransformerFactory transformerFactory = TransformerFactory.newInstance();
+			Transformer transformer = transformerFactory.newTransformer();
+			transformer.setOutputProperty(OutputKeys.INDENT, "yes");
+			transformer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "2");
+			DOMSource source = new DOMSource(doc);
+			
+			String saveLocation = "data/enchantments/"+name+".xml";
+			StreamResult result = new StreamResult(new File(saveLocation));
+			
+			transformer.transform(source, result);
+			
+		} catch (ParserConfigurationException pce) {
+			pce.printStackTrace();
+		} catch (TransformerException tfe) {
+			tfe.printStackTrace();
+		}
+		Main.game.setContent(new Response("Save", "", Main.game.getCurrentDialogueNode()));
+	}
+
+	public static void loadEnchant(String name)
+	{
+		if (isLoadEnchantAvailable(name))
+		{
+			File file = new File("data/enchantments/"+name+".xml");
+
+			if (file.exists())
+			{
+				try
+				{
+					DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+					DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+					Document doc = dBuilder.parse(file);
+					
+					// Cast magic:
+					doc.getDocumentElement().normalize();
+					
+					Element enchantment = (Element) doc.getElementsByTagName("enchantment").item(0);
+					Element itemEffects = (Element) enchantment.getElementsByTagName("itemEffects").item(0);
+					List<ItemEffect> effectsToBeAdded = new ArrayList<>();
+					for(int i=0; i<itemEffects.getElementsByTagName("effect").getLength(); i++)
+					{
+						Element e = ((Element)itemEffects.getElementsByTagName("effect").item(i));
+						ItemEffect itemEffect = ItemEffect.loadFromXML(e, doc);
+						if(itemEffect == null)
+						{
+							System.err.println("Warning: Failed to import ItemEffect");
+						}
+						else if(ingredient.getEnchantmentEffect() != itemEffect.getItemEffectType())
+						{
+							Main.game.flashMessage(Colour.GENERIC_BAD, "Enchantment is not valid for ingredient");
+							return;
+						}
+						else
+						{
+							effectsToBeAdded.add(itemEffect);
+						}
+					}
+					effects = effectsToBeAdded;
+				}
+				catch(Exception e)
+				{
+					e.printStackTrace();
+				}
+			}
+		}
+	}
+	public static boolean isLoadEnchantAvailable(String name)
```

Comment:
make a private static Optional<File> which returns empty optional if the file doesn't exist, and returns the file handle if it does. You can then use it in deleteEnchant, and in loadEnchant with ifPresent.